decidim-elections/app/commands/decidim/elections/admin/setup_election.rb

```diff
@@ -94,7 +94,10 @@ def questions_data
               weight: question.weight,
               max_selections: question.max_selections,
               title: flatten_translations(question.title),
-              description: flatten_translations(question.description),
+              # description: {flatten_translations(question.description)},
+              # the bulletin_board gem (ruby client) expects a description for the question
+              # as development is in a separate repository, let's send an empty content for the moment
+              description: {},
```

Comment:
What happens if we don't send this key? Do we get an error?

Reply:
Yes, if we don't want to send this info then we will need to modify the bulletin board as well (is required there under the name of `subtitle`).